pkg/sql/opt_catalog.go

```diff
@@ -892,7 +892,7 @@ func newOptTable(
 
 			invertedSourceColOrdinal, _ := ot.lookupColumnOrdinal(invertedColumnID)
 
-			// Add a inverted column that refers to the inverted index key.
+			// Add an inverted column that refers to the inverted index key.
 			invertedCol, invertedColOrd := newColumn()
 
 			// All inverted columns have type bytes.
```

Comment:
Should we change this comment to `// All inverted columns have type EncodedKey.`?